### PR TITLE
Feature/development

### PR DESCRIPTION
**Pull request by RokoVarano**
This project emulates tenextweb.com website. It features the following:

- 3 different view configurations for mobile (max-with: 768px), tablet (max-width: 1003px) and desktop. Size references are taken from the website.
- Sticky navbar.
- Responsive display types, including flexbox and grid. Conditional showing of items is managed with display as well 
- Responsive content, including item configuration and styles (like font-sizes)

TODO: DRY CSS.

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Invalid Code Review Request :disappointed: :cry: :x: 
## :handshake: :handshake: :handshake:  Hello @RokoVarano and @widzthedvloper :hand: 
:x: Your code review request has been marked as invalid because your repository doesn't look professional, please [read this lesson](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to know how you should make it professional.
### Some tips :traffic_light: 
:worried: You should add a custom readme file which follow microverse template
:worried: Make your pull request name meaningful, currently its name is Feature/development, that name is not meaningful at all. Make it descriptive; something like `add-nextweb-clone`... try things like that

**Comment by widzthedvloper** [comment]
we fixed all the linter error related to the CSS and HTML file. 
we had use sass in are program the only linter error that remain are inside the sass file we used.
Since we didn't have to study sass completely for this week we have issues fixing them (sass wasn't mandatory for this project we use it for the last exercise of the week which an introduction to processors )

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
-# TNW_clone2
-Second try for the TNW Clone
+# THE NEXT WEB-CLone
+This project is a clone of a thenextweb.com website. The page is responsive
+
+# Getting started :
+this website can be viewd better on the screen size of of 1920 X 1080 in Mozilla firefox browser.
```
Comment (arslanbisharat) [inline README.md]: - [x] Please use Grammarly to not get errors.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+
+- GitHub: [@githubhandle](https://github.com/widzthedvloper)
+- Twitter: [@twitterhandle](https://twitter.com/widzthedvloper)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/widzmarc-jean-nesly-phelle-252a26129/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/RokoVarano)
+- Twitter: [@twitterhandle](https://twitter.com/RodrigoIbacet11)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rodrigo-ibaceta-a8657611a/)
```
Comment (arslanbisharat) [inline README.md]: - [x] Add authors name please

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+
+## SHOW YOUR SUPPORT
+Give a * if you like the project
```
Comment (arslanbisharat) [inline README.md]: - [x] [Optional] You can add ⭐  by writing `:start: `in readme

File: README.md [hunk 4]
```diff
@@ -1,2 +1,40 @@
...
+
+## 📝 LICENSE
+This project as no license
```
Comment (arslanbisharat) [inline README.md]: - [x] Please remove this part or add a license

File: index.html [hunk 5]
```diff
@@ -3,11 +3,1249 @@
...
+                <li class="bars">
+                    <a href="#generic">
+                        <i class="fas fa-bars" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="cleaar"></div>
+    </div>
+</div>
```
Comment (arslanbisharat) [inline index.html]: - [x] Please enclose this part in the header tags

File: index.html [hunk 6]
```diff
@@ -3,11 +3,1249 @@
...
+                    <img src="assets/tnw-footer.svg" alt="">
+                </a>
+                © 2006–2020 The Next Web B.V. Made with &lt;3 in Amsterdam.
+                Powered by
+                <a href="#">
+                    <img src="assets/stackpath.svg" alt="">
+                </a>
+            </p>
+        </div>
+    </footer>
```
Comment (arslanbisharat) [inline index.html]: - [x] Please move footer outside main

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**
- [x] Please update your PR message. You are using the same message you used for the last review which got invalidated. 



- [x] In the future please make your commit messages descriptive. 

- [x] Please remove linter errors. 


- [x] Please use semantic HTML 
- only one <main> per page ;
- top-level element (in most cases) ;
- don't put your site's header/menu/footer (repeated content) in your <main>. can hold article header/footer if specific.

**Purpose**

The primary purpose of <main> is to map ARIA’s landmark role main** to an element in HTML. This will help screen readers and other assistive technologies understand where the main content begins. The W3C spec describes as representing:

Specs say

> The main content area consists of content that is directly related to or expands upon the central topic of a document or central functionality of an application.

### **Header/footer**
**Exclude site's header/footer/menu:**

excludes content that is repeated across a set of documents such as site navigation links, copyright information, site logos, and banners and search forms (unless the document or application's main function is that of a search form).

Continue to use `role='main'`
But as a side note to this article Steve Faulkner writes:

> You should still use the ARIA role until all browsers map the role to the <main> element.

- [x] Please use dry CSS. You have used many properties again and again. For example **display:flex**. You can learn more about dry CSS from [here](https://teamtreehouse.com/library/dry-css#:~:text=DRY%20stands%20for%20%22Don't,code%20in%20our%20style%20sheet.). 

# Desktop

### Navigation Bar

> Everything is fine here. Good work!

### Top 3 articles section

- [x] Plese change the from `background-color: antiquewhite` to` background-color:white`

**Clone**


**Code**


**Categories (Apps, Gear, Tech, etc) section**

- [x] Items should have a gap from each other. 

**Clone**


**Original**



**Footer**

- [x] Footer has too much vertical width(height). Please work on it. 



-------------------------------------------------------------

# Medium

- [x] Your whole webpage missing medium styling. You are mistakenly are given small version styling to the medium screen too.  Here are  some suggestions:

- On the navbar TWN logo appears at the leftmost part
- after the navbar in 3 article section TWN logo disappears
- Latest News section aligns 3 articles per row
- Latest funding section aligns 3 articles per row
- Categories (Apps, Gear, Tech, etc) section aligns 2 category boxes per row
- Latest deal section aligns 3 articles per row

---------------------------------------------------------------------

# Small

**Categories (Apps, Gear, Tech, etc) section** 

- [x] This should look like this 
 
But yours look like this






**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @RokoVarano and @RokoVarano, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR. Congratulations 🎊 👏, you did very well on this project. 👏👏👏  I appreciate your hard work in replicating thenextweb.com.

Now, you can move ahead.

Please don't forget to merge to master. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by layzrd** [comment]
Hi @RokoVarano and @widzthedvloper, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏 You did well on this project. I appreciate your efforts to achieve this milestone 🦍. 

But, this project is already approved,  and it will mark as complete soon, so you don't have to create another code review request to microverse.

Thanks for your patience.

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by Srjnnnn** [comment]
Dear @widzthedvloper. I am not sure what you've mixed and what makes you struggle, however, I can not approve a project that is already approved. So I marked your submission as invalid and I highly recommend you to reach out to student success and explain the urgency of your problem. I am sure that they'll have the right resources to help you. 

Happy coding :100: 
Eyüp Sercan UYGUR

**Pull request closed without merging**